mods/gen2/items.js

```diff
@@ -30,6 +30,7 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects.",
```

Comment:
Also note that PS's stage 3 is actually stage 2 since the most common understanding of critical hit stages is that normal moves start at stage 0.